conf/nc6s_v3.conf

```diff
@@ -0,0 +1,37 @@
+# NHC Configuration File
+#
+# Lines are in the form "<hostmask>||<check>"
+# Hostmask is a glob, /regexp/, or {noderange}
+# Comments begin with '#'
+#
+# This file was automatically generated by nhc-genconf
+# Fri Nov 17 22:50:44 UTC 2023
+#
+
+#######################################################################
+###
+### NHC Configuration Variables
+###
+# ncv3nhcvm || export MARK_OFFLINE=1 NHC_CHECK_ALL=0
+
+
+#######################################################################
+###
+### Hardware checks
+###
+ * || check_hw_cpuinfo 1 6 6
+ * || check_hw_physmem 110GB 110GB 3%
```

Comment:
NIT: Some SKUs are with kB, some are with MB, and GB.
Do we want unify the unit?

Reply:
Yea that's a bit weird. This check was auto generated by NHC. Let me think about this one, I will probably not address in this PR.